src/Simplex/Chat/Store/Messages.hs

```diff
@@ -1083,37 +1084,63 @@ getDirectChatAround_ db user ct@Contact {contactId} aroundItemId count search =
   let chatItems = beforeChatItems <> [middleChatItem] <> afterChatItems
   pure $ Chat (DirectChat ct) chatItems stats
 
-getGroupChat :: DB.Connection -> VersionRangeChat -> User -> Int64 -> ChatPagination -> Maybe String -> ExceptT StoreError IO (Chat 'CTGroup)
+getDirectChatInitial_ :: DB.Connection -> User -> Contact -> Int -> ExceptT StoreError IO (Chat 'CTDirect, ChatLandingSection)
+getDirectChatInitial_ db user@User {userId} ct@Contact {contactId} count = do
+  firstUnreadItem <- liftIO getDirectChatMinUnreadItemId_
+  case firstUnreadItem of
+    Just firstUnreadItemId -> do
+      chat <- getDirectChatAround_ db user ct firstUnreadItemId count ""
```

Comment:
search not passed because chat is always opened without it? probably ok, but could be setting trap for future.. maybe at least worth throwing error in Chat on api call - validate that if it's initial search should be Nothing. or even simpler to pass it here, it shouldn't hurt since in current real scenario it'll be empty